ref/randombytes.c

```diff
@@ -38,12 +38,12 @@ void randombytes(uint8_t *out, size_t outlen) {
   if(!CryptReleaseContext(ctx, 0))
     abort();
 }
-#elif defined(__linux__) && defined(SYS_getrandom)
+#elif defined(__linux__) && defined(__GLIBC__) && ((__GLIBC__ > 2) || (__GLIBC_MINOR__ > 24))
 void randombytes(uint8_t *out, size_t outlen) {
   ssize_t ret;
 
   while(outlen > 0) {
-    ret = syscall(SYS_getrandom, out, outlen, 0);
+    ret = getrandom (out, outlen, 0);
```

Comment:
this isn't using `syscall` by accident; it allows for use of `getrandom` on e.g. older versions of Ubuntu where `getrandom()` isn't available but the syscall does work.